packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx

```diff
@@ -0,0 +1,173 @@
+import { Component, h, JSX, Prop, Element, Fragment } from '@stencil/core';
+import { trackComponent } from '@utils/tracking/usage';
+import {
+  GuxAvatarStatus,
+  GuxAvatarSize,
+  GuxAvatarAccent
+} from './gux-avatar.types';
+import { hasSlot } from '@utils/dom/has-slot';
+import { logWarn } from '@utils/error/log-error';
+
+/**
+ * @slot image - Headshot photo.
+ */
+
+@Component({
+  styleUrl: 'gux-avatar.scss',
+  tag: 'gux-avatar-beta',
+  shadow: true
+})
+export class GuxAvatar {
+  @Element()
+  root: HTMLElement;
+
+  @Prop()
+  status: GuxAvatarStatus = 'available';
+
+  @Prop()
+  size: GuxAvatarSize = 'large';
+
+  /**
+   * Shows a status ring around the avatar
+   */
+  @Prop()
+  statusRing: boolean = false;
+
+  @Prop()
+  name!: string;
```

Comment:
Would it be better just to call this `initals` or something since even if you were to put in your full name it will just convert to the `initials` of the user.

Reply:
We want to encourage the use of the full name here for accessibility & to generate a more unique accent.
We don't want people passing in the initials directly.